src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -312,6 +303,15 @@ await CommandLineOptionsValidator.CheckForObsoleteOptionsAsync(
         // to file disc also the banner, so at this point we need to have all services and configuration(result directory) built.
         await DisplayBannerIfEnabledAsync(loggingState, proxyOutputDevice, testFrameworkCapabilities, testApplicationCancellationTokenSource.CancellationToken).ConfigureAwait(false);
 
+        // Check for obsolete options and display warnings after the banner
+        await CommandLineOptionsValidator.CheckForObsoleteOptionsAsync(
+            loggingState.CommandLineParseResult,
+            commandLineHandler.SystemCommandLineOptionsProviders,
+            commandLineHandler.ExtensionsCommandLineOptionsProviders,
+            proxyOutputDevice,
+            commandLineHandler,
+            testApplicationCancellationTokenSource.CancellationToken).ConfigureAwait(false);
+
```

Comment:
This change is somehow unrelated to this PR, it's just than having a proper case, I have noticed the placement of the warning message is not good. I'll extract in a different PR.